main/zoe/api/zoe-contract-facet.md

```diff
@@ -101,6 +99,23 @@ Get the Zoe `inviteIssuer`.
 const inviteIssuer = await E(zoe).getInviteIssuer();
 ```
 
+## zcf.getBrandForIssuer(issuer)
+- `issuer` `{Issuer}`
+- Returns `{Brand}`
+
+Synchronously returns the `brand` of the `issuer` argument
```

Comment:
"synchronously" is not different for this and any other ZCF call. Rather than adding it everywhere, I'd drop it here. If you think it's important, I'd find a global place to mention that all calls to ZCF are synchronous.

Reply:
Copied from a code comment, will drop.